src/Slide.js

```diff
@@ -1,64 +1,31 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import cx from 'classnames';
-import constants from './constants';
 
-class Slide extends Component {
-  constructor(props) {
-    super(props);
-    this.renderCaption = this.renderCaption.bind(this);
-  }
-
-  render() {
+const Slide = ({ src, children }) => {
+  if (typeof src === 'string') {
     return (
       <li>
-        <img src={this.props.src} />
-        {this.renderCaption()}
+        <img src={src} alt="" />
+        {children}
+      </li>
+    );
+  } else {
+    return (
+      <li>
+        {src}
+        {children}
```

Comment:
I don't quite understand this, `<li>{src}{children}</li>`

Reply:
why would `src` not be a string?

Reply:
Going to change this props from `src` to `image` and add an example to make it clear ASAP

Reply:
@alextrastero Done, please take a look at it whenever you wish